doozer/doozerlib/cli/images_streams.py

```
@@ -615,134 +615,134 @@ def add_localdev_repo_profile(profile):
+            cmd = f"oc apply -f {output}"
             if as_user:
-                cmd += f' --as {as_user}'
+                cmd += f" --as {as_user}"
             exectools.cmd_assert(cmd, retries=3)
         else:
-            print('No buildconfigs were generated; skipping apply.')
+            print("No buildconfigs were generated; skipping apply.")
 
 
 def calc_parent_digest(parent_images):
     m = hashlib.md5()
```

Ruff (0.15.0)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)